RiskIdent/traefik-tls-headers-plugin
# Add TLS Cipher header

Ticket: OP-2637

## Description

This plugin takes TLS information and stores it in the request headers.
This PR sets the base to do so and takes care of setting the cipher used by the connection.

## How has this been tested

1. Execute the headers reader web service `make start_headers_reader`
2. Execute traefik with the plugin locally loaded `make testcontainer`
3. Make a request to traefik `curl https://localhost -k`
The response should contain a non-empty `X-Tls-Cipher` header.
